src/Microsoft.Azure.WebJobs.Extension.Storage/XStorageAccountProvider.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Microsoft.WindowsAzure.Storage.Table;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs
+{
+    // Storage Extension's abstraction for a storage provider. 
+    // $$$ Replaced DefaultStorageProvider, IStorageAccountProvider.
+    public class XStorageAccountProvider
```

Comment:
IStorageAccountProvider used to be internal - I think this should still be internal? Also, not sure about the X prefix - can you rename to just StorageAccountProvider?

Reply:
This was made public to allow for replacement. Part of @MikeStall's storage changes, so he can comment here.